src/root/longdouble.c

```diff
@@ -50,7 +50,9 @@ bool initFPU()
     int old_cw = _control87(_MCW_EM | _PC_64  | _RC_NEAR,
                             _MCW_EM | _MCW_PC | _MCW_RC);
 #endif
+#if _MSC_VER < 1700
```

Comment:
_set_output_format still compiles and links with VS2013. The C runtime changed very little until VS2015.